.github/workflows/tests.yml

```diff
@@ -45,6 +45,9 @@ jobs:
         django-version:
           - 4.2.*
           - 5.0.*
+        graphql-core-version:
+          - 3.3.0a5
+          - 3.2.3
```

Comment:
Maybe `3.2.*`, just to make sure we get the latest patch release, like we do for django?

Not sure if `3.3.*` would work as it is a pre-release, and usually those have to be totally hardcoded